tensorflow/docs_src/api_guides/python/contrib.signal.md

```diff
@@ -34,8 +34,8 @@ with inner structure (e.g. a spectrogram):
 magnitude_spectrograms = tf.abs(tf.contrib.signal.stft(
```

Comment:
Thanks! Could you please also fix the 127 in the above comment?

Reply:
Done.